pyproject.toml

```diff
@@ -74,3 +74,195 @@ multi_line_output = 3
 include_trailing_comma = true
 combine_as_imports = true
 skip = "src/twisted/logger/__init__.py,src/twisted/internet/reactor.py"
+
+
+[tool.tox]
+legacy_tox_ini = """
```

Comment:
This breaks syntax highlighting of tox.ini for no clearly-articulated gain. I am firmly :-1: on this change.